src/scenes/Partners/Detail/PartnerDetail.graphql

```diff
@@ -85,6 +85,30 @@ fragment partnerOwnDetails on Partner {
       ...DisplayLocation
     }
   }
+  languageOfWiderCommunication {
+    canRead
+    canEdit
+    value {
+      ...LanguageListItem
+      ...LanguageLookupItem
```

Comment:
`<LanguageListItem>` is not used here, so don't use its fragment.
Declare the fields read inline, or creating a new fragment is fine too.

`LanguageLookupItem` is fine here, since it's used in the form. But we shouldn't rely on what fields it asks for either for my above statement.